src/json-reporter.js

```diff
@@ -103,8 +104,14 @@ function reporterFactory() {
         const videoAssets = testRunInfo.videos
           .filter(idFilter)
           .map((v) => this._getAsset(v.videoPath, testName));
-        const testStartTime = this.startTimes.get(testRunInfo.testId);
 
+        const sauceAttachments =
+          testRunInfo.reportData[browser.testRunId]?.find(
+            (item) => item.sauceAttachments,
```

Comment:
`t.report` takes args of type `...args: any[]` so I'm wondering if we want to at least have null(ish) check on `item` in this find predicate. For example, I could see some doing something silly like 

```ts
t.report(variableThatIsUnexpectedlyNull, { sauceAttachments: [] })
```